configuration/config-txt/boot.md

```diff
@@ -113,7 +113,11 @@ On the Raspberry Pi 4B, if this value is set to `0` then the interrupts will be
 
 ## force_eeprom_read
 
-Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup.
+Set this option to `0` to prevent the firmware from trying to read an I2C HAT EEPROM (connected to pins ID_SD & ID_SC) at powerup. See also `disable_poe_fan`.
+
+## disable_poe_fan
+
+Set this option to `1` to prevent control of the PoE HAT fan through I2C on pins ID_SD & ID_SC). Without this, a probe on the I2C bus will happen at startup, even when not using the PoE HAT.
```

Comment:
Seems to be a stray `)` ?

Reply:
Fixed, sorry for the noise... Seems I over-estimated the trivialness of this change and under-estimated the need for doing a proper review of my own commit... :-(